docs/readme.md

```diff
@@ -1,6 +1,6 @@
 # Brigade Docs
 
-* Browse the [docs on GitHub](https://github.com/brigadecore/brigade/tree/master/docs/content)
+* Browse the [docs on GitHub](https://github.com/brigadecore/brigade/tree/v1/docs/content)
 * Browse the [docs website at docs.brigade.sh](https://docs.brigade.sh)
```

Comment:
This URL should be updated to `https://v1--brigade-docs.netlify.app/`, right?

Reply:
Good catch.

Reply:
Fixed. And fyi, longer term, I will also work out a nicer URL than `https://v1--brigade-docs.netlify.app/`.